lib/tictactoe.rb

```diff
@@ -20,6 +20,16 @@ def initialize (game_io, player1, player2)
   end
 
   def play_game
+    begin
     @game_loop.game_loop
+    @board.new_game
```

Comment:
I changed the code so that it creates a new `GameBoard` object. I had to add `attr_accessor :board` to both the `GameLoop` and `GameScreen` classes as they need a reference to the `GameBoard` object. So when `TicTacToe` creates a new `GameBoard` it can update the reference in those other to classes to point to the current board.